src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -63,6 +62,8 @@ public async Task<IActionResult> Login(LoginViewModel viewModel)
             var returnUrl = viewModel.ReturnUrl.ToString();
 
             if (signInResult.IsTrustedReturnUrl)
```

Comment:
Not related to this PR, but I thought we agreed to always have braces around if statements

Reply:
We agreed not to strictly enforce one way or the other, except when the statement block spans multiple lines in which case it should always have braces.

Perhaps I should start putting a document together rather than waiting for our next meeting...

Reply:
👍